docs/reference/api.md

```diff
@@ -1455,7 +1480,37 @@ const account = await fcl.account("0x1d007d755706c469");
 
 ---
 
-## `latestBlock`
+## `block`
+
+A pre-built interaction that returns the latest block (optionally sealed or not), by id, or by height.
+
+#### Arguments
+
+| Name     | Type    | Default | Description                                                                    |
+| -------- | ------- | ------- | ------------------------------------------------------------------------------ |
+| `sealed` | boolean | false   | If the latest block should be sealed or not. See [block states](#interaction). |
+| `id`     | boolean | string  | ID of block to get. See [block states](#interaction).                          |
+| `height` | boolean | int     | Height of block to get. See [block states](#interaction).                      |
+
+#### Returns
+
+| Type                        | Description                       |
+| --------------------------- | --------------------------------- |
+| [BlockObject](#blockobject) | A JSON representation of a block. |
+
+#### Usage
+
+```javascript
+import * as fcl from "@onflow/fcl";
+await fcl.block() // get latest finalized block
+await fcl.block({sealed: true}) // get latest sealed block
+await fcl.block({id: 'abc'}) // get block by id
```

Comment:
Let's use a real block id - I know it won't be usable by the time the user goes to use it but I think it's good to be consistent with the ID format.

Reply:
same with height